.github/workflows/tests.yml

```
@@ -17,40 +17,149 @@ jobs:
+          if [ -z "$found" ]; then echo "Obfucc not found; searched: ${candidates[*]}"; exit 1; fi
+          echo "Found built obfucc at: $found"
+          mkdir -p build
+          cp "$found" build/obfucc
+          chmod +x build/obfucc
+          echo "Copied $found -> build/obfucc"
         shell: bash
 
-      - name: Run regression tests
+      - name: Add obfucc to PATH (Unix)
+        if: runner.os != 'Windows'
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:35: Double quote to prevent globbing and word splitting

(shellcheck)